Add new set of predictor variables in config/datasets for the wind dataset

epavel1 opened the pull request:
Adds a new set of predictor variables in config/datasets for the wind dataset. This new set of predictor variables is a extended version of the original predictor variables, in which the variables 'z_pl850_in' and 'z_pl700_in' were added. This decision was made, after a ablation study has shown a slight but consistent improvement by adding these two variables.

epavel1 committed:
Add new set of predictor variables in config/datasets for the wind dataset

mlangguth89 reviewed (commented):
@epavel1 Can you please ensure that there is only one configuration-file for the wind downscaling dataset named `config_ds_wind.json`?
Furthermore, you may introduce a key 'static_predictors' to the dataset-config. In this case, it would look as follows:
```
[...]
"static_predictors": ["z_in", "lsm_in", "hsurf_tar", "fr_land_tar"],
[...]
```
Remember to adapt `predictors` accordingly. 
Also note that the `var_tar2in`-key has recently become deprecated (see [here](https://github.com/mlangguth89/downscaling_benchmark/blob/c0227d072c871506c739418810e4ef85ea83a601/handle_data/handle_data_class.py#L305)), whereas the `varname_z`-key is only effective when the model has an active `z_branch`-key (see [here](https://github.com/mlangguth89/downscaling_benchmark/blob/c0227d072c871506c739418810e4ef85ea83a601/handle_data/handle_data_class.py#L288)).

epavel1 committed:
Rename config_ds_wind_extended.json to config_ds_wind.json

Rename wind config_ds_wind_extended.json file to config_ds_wind.json in order to provide a better understanding of the file

epavel1 committed:
Update config_ds_wind.json

epavel1 committed:
Delete config/datasets/config_ds_wind_deepru.json
